src/__swaps__/safe-math/SafeMath.ts

```diff
@@ -202,6 +202,32 @@ export function modWorklet(num1: string | number, num2: string | number): string
   return formatResultWorklet(result);
 }
 
+// return significant decimals in fractional portion of number
+export function significantDecimalsWorklet(num: string | number): number {
+  'worklet';
+  if (num === 0) {
+    return 0;
+  }
+
+  // Convert the number to a string to handle large numbers and fractional parts
+  const numStr = num.toString();
+
+  // Split the number into integer and fractional parts
+  const [_, fractionalPart] = numStr.split('.');
+
+  // Handle fractional parts
+  if (fractionalPart) {
```

Comment:
I created a new function here for now as the `orderOfMagnitudeWorklet` which is used in other places doesn't exactly capture what we sometimes care about for counting significant decimals. e.g, in this function, I'm curious what the first significant decimal is, regardless of what's to the left of the decimal.